README.md

```diff
@@ -55,6 +62,26 @@ test('very important test happening here', function(assert) {
 
 Both the global sinon object and the `this.sandbox` convenience property point to the same, test-specific instance of a sinon sandbox.
 
+### QUnit `hooks` API
+
+To use, import the setup method from within your test file and execute it.
+
+```js
+import setupSinonSandbox from 'ember-sinon-sandbox/test-support';
+
+...
+
+module('my module', {
```

Comment:
should be `module('my module', function(hooks) {`

Reply:
Sorted!